extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
two spaces for one indentation in pom files

Reply:
The formatter turns them into tabs 😕